src/PhpSpreadsheet/Writer/Xlsx.php

```diff
@@ -406,6 +412,8 @@ public function save($pFilename)
                 }
                 @unlink($pFilename);
             }
+            return true;
```

Comment:
IWriter does not specify any return values. We should not return anything.

Reply:
So, How can we check if file is created successfully or not by $writer->save()? @PowerKiKi 
For Eg. If I want to continue execution of my script *only if file is created*.

Lets say:
```try {
if ($writer->save('hello world.xlsx') === true)
    echo 'File Created'; // Insert into database
}
catch (WriterException $e) {
    echo $e->getMessage();
    // die('File could not be created');
}
```

Reply:
This would be enough:

```php
try {
    $writer->save('hello world.xlsx');
    echo 'File Created'; // Insert into database
}
catch (WriterException $e) {
    echo $e->getMessage();
    // die('File could not be created');
}
```